### PR TITLE
release v18

### PR DESCRIPTION
**Pull request by mjudeikis**
```release-note
NONE
```
Add changelog. Pin containers

### CODE REVIEW

**Comment by mjudeikis** [comment]
https://github.com/openshift/release/pull/9434/files Needs to go in

File: pluginconfig/pluginconfig-311.yaml [hunk 1]
```diff
@@ -265,14 +265,14 @@ versions:
       genevaTDAgent: osarpint.azurecr.io/acs/td-agent:master.20190228.1
       logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod03022020
       metricsServer: registry.access.redhat.com/openshift3/ose-metrics-server:v3.11.200
-      azureControllers: quay.io/openshift-on-azure/ci-azure:latest
```
Comment (ehashman) [inline pluginconfig/pluginconfig-311.yaml:268]: There's no new VM image/OpenShift version? We need to include that for security fixes.

Comment (mjudeikis) [reply pluginconfig/pluginconfig-311.yaml:268]: I wanted to play it safe :) updated and started new image publish as we didn't had one

File: pluginconfig/pluginconfig-311.yaml [hunk 2]
```diff
@@ -231,48 +231,48 @@ versions:
...
+      prometheusConfigReloader: registry.access.redhat.com/openshift3/ose-prometheus-config-reloader:v3.11.219
+      prometheusOperator: registry.access.redhat.com/openshift3/ose-prometheus-operator:v3.11.219
+      registry: registry.access.redhat.com/openshift3/ose-docker-registry:v3.11.219
+      registryConsole: registry.access.redhat.com/openshift3/registry-console:v3.11.219
+      router: registry.access.redhat.com/openshift3/ose-haproxy-router:v3.11.219
+      serviceCatalog: registry.access.redhat.com/openshift3/ose-service-catalog:v3.11.219
+      templateServiceBroker: registry.access.redhat.com/openshift3/ose-template-service-broker:v3.11.219
+      webConsole: registry.access.redhat.com/openshift3/ose-web-console:v3.11.219
+      format: registry.access.redhat.com/openshift3/ose-${component}:v3.11.219
       httpd: registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-110
```
Comment (ehashman) [inline pluginconfig/pluginconfig-311.yaml:261]: 2.4-115 is available

File: pluginconfig/pluginconfig-311.yaml [hunk 3]
```diff
@@ -231,48 +231,48 @@ versions:
...
       masterEtcd: registry.access.redhat.com/rhel7/etcd:3.2.26-37
```
Comment (ehashman) [inline pluginconfig/pluginconfig-311.yaml:262]: 3.2.28-8 is available

File: pluginconfig/pluginconfig-311.yaml [hunk 4]
```diff
@@ -234,45 +234,45 @@ versions:
...
-      tlsProxy: quay.io/openshift-on-azure/ci-azure:latest
+      metricsServer: registry.access.redhat.com/openshift3/ose-metrics-server:v3.11.219
+      azureControllers: quay.io/openshift-on-azure/ci-azure:v18.0
+      aroAdmissionController: quay.io/openshift-on-azure/ci-azure:v18.0
+      canary: quay.io/openshift-on-azure/ci-azure:v18.0
+      etcdBackup: quay.io/openshift-on-azure/ci-azure:v18.0
+      metricsBridge: quay.io/openshift-on-azure/ci-azure:v18.0
+      startup: quay.io/openshift-on-azure/ci-azure:v18.0
+      sync: quay.io/openshift-on-azure/ci-azure:v18.0
+      tlsProxy: quay.io/openshift-on-azure/ci-azure:v18.0
```
Comment (ehashman) [inline pluginconfig/pluginconfig-311.yaml:275]: Image needs to be pushed to ACR, not quay :)

**Comment by mjudeikis** [comment]
/override upgrade-v18.0
no point testing upgrade to itself.

**Pull request merged**